great_expectations/jupyter_ux/expectation_explorer.py

```diff
@@ -1,414 +1,1376 @@
-            'expect_column_most_common_value_to_be_in_set': ['value_set', 'ties_okay'],
+            ####
             'expect_column_pair_values_to_be_equal': ['ignore_row_if'],
-            'expect_column_pair_values_A_to_be_greater_than_B': ['or_equal', 'allow_cross_type_comparisons', 'ignore_row_if'],
+            'expect_column_pair_values_A_to_be_greater_than_B': ['or_equal', 'ignore_row_if', 'allow_cross_type_comparisons'],
             'expect_column_pair_values_to_be_in_set': ['value_pairs_set', 'ignore_row_if'],
-            ####
+            'expect_multicolumn_values_to_be_unique': ['ignore_row_if'],
             'expect_column_values_to_be_of_type': ['type_', 'mostly'],
             'expect_column_values_to_be_in_type_list': ['type_list', 'mostly'],
-            'expect_multicolumn_values_to_be_unique': ['ignore_row_if'],
+            'expect_column_kl_divergence_to_be_less_than': ['partition_object', 'threshold', 'internal_weight_holdout', 'tail_weight_holdout'],
             'expect_column_chisquare_test_p_value_to_be_greater_than': ['partition_object', 'p'],
             'expect_column_bootstrapped_ks_test_p_value_to_be_greater_than': ['partition_object', 'p', 'bootstrap_samples', 'bootstrap_sample_size'],
             'expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than': ['distribution', 'p_value', 'params'],
+            '_expect_column_values_to_be_in_type_list__aggregate': ['type_list'],
+            '_expect_column_values_to_be_in_type_list__map': ['type_list']
         }
         self.kwarg_widget_exclusions = [
             'column', 'result_format', 'include_config']
-        # debug_view is for debugging ipywidgets callback functions
-        self.debug_view = widgets.Output(layout={'border': '3 px solid pink'})
+        self.min_max_subtypes = {
+            'integer': {
+                'positive': [
+                    'expect_column_unique_value_count_to_be_between',
+                    'expect_column_value_lengths_to_be_between',
+                    'expect_table_row_count_to_be_between'
+                ],
+                'unbounded': ['expect_column_median_to_be_between'],
+            },
+            'float': {
+                'unit_interval': ['expect_column_proportion_of_unique_values_to_be_between'],
+                'unbounded': [
+                    'expect_column_mean_to_be_between',
+                    'expect_column_stdev_to_be_between',
+                    'expect_column_sum_to_be_between'
+                ]
+            },
+            'mixed': {
+                'unbounded': [
+                    'expect_column_values_to_be_between',
+                    'expect_column_min_to_be_between',
+                    'expect_column_max_to_be_between'
+                ]
+            }
+        }
+        self.debug_view = widgets.Output()
+        self.styles = {
+            'description_width': {'description_width': '150px'}
+        }
```

Comment:
I bet this whole list of parameters is going to end up being super useful in other places.

For example, it could be super helpful for creating metrics when profiling. Pinging @eugmandel to share state...